src/lib389/lib389/cli_ctl/tls.py

```diff
@@ -33,34 +38,102 @@ def import_client_ca(inst, log, args):
     tls.add_cert(nickname=nickname, input_file=cert_path)
     tls.edit_cert_trust(nickname, "T,,")
 
+
 def import_ca(inst, log, args):
+    if not os.path.isfile(args.cert_path):
+        raise ValueError(f'Certificate file "{args.cert_path}" does not exist')
+
     tls = NssSsl(dirsrv=inst)
-    cert_path = args.cert_path
-    nickname = args.nickname
-    if nickname.lower() == CERT_NAME.lower() or nickname.lower() == CA_NAME.lower():
-        log.error("You may not import a CA with the nickname %s or %s" % (CERT_NAME, CA_NAME))
-        return
-    tls.add_cert(nickname=nickname, input_file=cert_path)
-    tls.edit_cert_trust(nickname, "C,,")
+    try:
```

Comment:
The code here and in `src/lib389/lib389/cli_conf/security.py` is very similar.

I think it may be worth encapsulating the code to `NssSsl()` class:
https://github.com/389ds/389-ds-base/blob/main/src/lib389/lib389/nss_ssl.py#L53

`add_cert` is already there and it'll be natural to have `add_cert_bundle` (to something), IMO.

What do you think?